helm/chaos-mesh/templates/_helpers.tpl

```diff
@@ -113,5 +113,30 @@ Define the webhook's name
 Define the prefix of
 */}}
 {{- define "registry-prefix" -}}
-{{if .Values.registry}}{{.Values.registry}}/{{end}}
+{{if .Values.images.registry}}{{.Values.images.registry}}/{{end}}
+{{- end -}}
+
+{{/*Define the image for chaos-controller-manager*/}}
+{{- define "chaos-controller-manager.image" -}}
+{{.Values.controllerManager.image.registry | default .Values.images.registry}}/{{.Values.controllerManager.image.repository}}:{{.Values.controllerManager.image.tag |  default .Values.images.tag}}
+{{- end -}}
+
+{{/*Define the image for chaos-daemon*/}}
+{{- define "chaos-daemon.image" -}}
+{{.Values.images.registry | default .Values.images.registry}}/{{.Values.chaosDaemon.image.repository}}:{{.Values.chaosDaemon.image.tag | default .Values.images.tag}}
```

Comment:
typo, it should starts with `{{ .Values.xxx.images.registry .....` here 👀

Reply:
fixed in 23d2e2ff